proposals/lip-0045.md

```diff
@@ -655,110 +688,59 @@ An entry for the mainchain is present by default, where:
 
 The `appendToInboxTree` function appends a new element to the underlying Merkle tree of the inbox of a chain account.
 
-##### Parameters
-
-This function has the following input parameters in the order given below:
-
-* `chainID`: The partner chain ID.
-* `appendData`: A bytes value.
-
-##### Returns
-
-A boolean indicating the success of the function execution.
-
-##### Execution
-
 ```python
-appendToInboxTree(chainID, appendData):
+def appendToInboxTree(chainID: ChainID, appendData: bytes) -> None:
     # Update root, appendPath, and size
-    let inboxTree be the underlying Merkle tree of channel(chainID).inbox
+    inboxTree = Merkle tree of channel(chainID).inbox
```

Comment:
Here I assume the whole Merkle tree is not necessary, rather the append path, root and size is sufficient. Maybe this could be made more clear.

Reply:
yes. Here we basically instantiate a 'light Merkle tree' from the root, append path, and size (appendix B in LIP31). I think this formulation is fine, but if you have a better suggestion I can change it

Reply:
Maybe 

>  inboxTree = partial Merke tree with properties from channel(chainID).inbox

or simply 

> inboxTree = channel(chainID).inbox

The phrase "Merkle tree of" sounds like an operation is performed, where really it is just coping the data. What is a bit odd that directly below you assign each property separately whereas here you do it all in one line.

Reply:
We do perform an operation tho, we have to instatiate a Merkle tree object, and in particular call the method `update` on it. That's why I think `inboxTree = channel(chainID).inbox` is incorrect.

Reply:
You mean the `append` call we make in the next line? How about the following suggestion then: 

> inboxTree = partial Merke tree with properties from channel(chainID).inbox

Reply:
ok, fixed